docs/tutorials/rootless_tutorial.md

```diff
@@ -6,14 +6,14 @@ Prior to allowing users without root privileges to run Podman, the administrator
 
 ## cgroup V2 support
 
-The cgroup V2  Linux kernel feature allows the user to limit the amount of resources a rootless container can use.  If the Linux distribution that you are running Podman on is enabled with  cgroup V2 then you might need to change the default OCI Runtime.  The default runtime `runc` does not currently work with cgroup V2 enabled systems, so you have to switch to the alternative OCI runtime `crun`.
+The cgroup V2  Linux kernel feature allows the user to limit the amount of resources a rootless container can use.  If the Linux distribution that you are running Podman on is enabled with  cgroup V2 then you might need to change the default OCI Runtime.  If you are using `runc` as your default runtime, it currently does not work with cgroup V2, you have to switch to the alternative OCI runtime `crun`.
```

Comment:
No longer true with a recent `runc`. Fedora's doesn't work but if you build from master it's fine.

Reply:
Changed to:

```
Some older versions of `runc` do not work with cgroup V2, you might have to switch to the alternative OCI runtime `crun`.
```